presto-main/src/main/java/io/prestosql/dispatcher/DispatchManager.java

```diff
@@ -132,6 +132,9 @@ public QueryManagerStats getStats()
     /**
      * Generates the next query identifier.
      *
+     * The id format is:
+     * - YYYYMMdd_HHmmss_generic-count_coordinator-id
```

Comment:
I think we could use {} in here:
```suggestion
     * - `{YYYYMMdd_HHmmss}_{generic-count}_{coordinator-id}
```
It seems better to read